.jscs.json

```diff
@@ -8,10 +8,16 @@
         "beforeOpeningRoundBrace": true
     },
     "requireMultipleVarDecl": true,
-    "disallowLeftStickedOperators": ["?", "-", "+", "/", "*", "=", "==", "===", "!=", "!==", ">", ">=", "<", "<="],
-    "disallowRightStickedOperators": ["?", "/", "+", "*", ":", "=", "==", "===", "!=", "!==", ">", ">=", "<", "<="],
-    "requireRightStickedOperators": ["!"],
-    "requireLeftStickedOperators": [","],
+    "requireSpaceBeforeBinaryOperators": [
```

Comment:
Thanks for updating this. I'm probably going to want to move on to eslint eventually.